activation/post_verifier_test.go

```diff
@@ -50,5 +50,39 @@ func TestOffloadingPostVerifier(t *testing.T) {
 
 func TestPostVerfierDetectsInvalidProof(t *testing.T) {
 	verifier := activation.NewPostVerifier(activation.PostConfig{}, log.NewDefault(t.Name()))
-	require.Error(t, verifier.Verify(&shared.Proof{}, &shared.ProofMetadata{}))
+	require.Error(t, verifier.Verify(context.Background(), &shared.Proof{}, &shared.ProofMetadata{}))
+}
+
+func TestPostVerifierVerifyAfterStop(t *testing.T) {
```

Comment:
Can we also have a test where `Verify` doesn't return unless its context is canceled and then we cancel the PostVerifier to see if the signal is propagated?